### PR TITLE
add hand interaction concept

### PR DESCRIPTION
**Pull request by aprilgittens**

### CODE REVIEW

File: lessons/lesson-3/concepts.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Concepts
 
+Hand interaction is a very natural way to interact with 3D models. As an interaction model, the use of hands is great for new users coupled with voice, eye tracking or head gaze. There is a low learning curve and consistent UX across hand tracking.
+
+Hand interaction may be suitable for your mixed reality application if any of the scenarios below apply to your mixed reality application:
```
Comment (Yonet) [inline lessons/lesson-3/concepts.md]: This could be it's own question, as well as line 11.

File: lessons/lesson-3/concepts.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # Concepts
 
+Hand interaction is a very natural way to interact with 3D models. As an interaction model, the use of hands is great for new users coupled with voice, eye tracking or head gaze. There is a low learning curve and consistent UX across hand tracking.
```
Comment (Yonet) [inline lessons/lesson-3/concepts.md]: I(the user) am not sure what is hand interaction and interaction model is. Please explain. 

>  There is a low learning curve and consistent UX across hand tracking.

Not sure what that means.

File: lessons/lesson-3/concepts.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+
+Hand interaction may be suitable for your mixed reality application if any of the scenarios below apply to your mixed reality application:
+
+- You want users to touch holograms and perform precision holographic manipulations.
+- You want users to have the lowest learning curve and most intuitive interactions.
+- Your users are generally mobile, covering a large space or moving between spaces.
+
+When implementing hands as an interaction model for your mixed reality application, consider the following affordances and behaviors to help new users:
+
+- To indicate that hand interaction is working, add hand mesh, a fingertip affordance or hand rays to your mixed reality application.
```
Comment (Yonet) [inline lessons/lesson-3/concepts.md]: Not sure what fingertip affordance is, please explain.

**Review by Yonet — COMMENTED** [review]
Hi April, can you follow the structure of questions and answers please, as shown in this example: https://github.com/Yonet/AzureMixedRealityDocs/tree/master/lessons/lesson-7/concepts.

Also, assuming the user hasn't heard of any of the terms might be helpful. Imagine that you are explaining to your friends from your previous career. Adding gifs and images helps as well.

Thank you for the quick PR.

File: lessons/lesson-3/concepts.md (hunk 1 above)
Comment (aprilgittens) [reply lessons/lesson-3/concepts.md]: I can pull out lines 5-9 into it's own question.

And I can pull out lines 11-15 into it's own question.

**Comment by aprilgittens** [comment]
@Yonet changed titles to lowercase and edited the concepts. Please let me know if anything here needs to be updated.

**Pull request closed without merging**